--- airflow.cfg.orig	2018-07-12 12:31:36.540874731 +0000
+++ airflow.cfg	2018-07-12 12:24:25.505209489 +0000
@@ -23,7 +23,7 @@
 # The amount of parallelism as a setting to the executor. This defines
 # the max number of task instances that should run simultaneously
 # on this airflow installation
-parallelism = 1500
+parallelism = 5000
 
 # Whether to load the examples that ship with Airflow. It's good to
 # get started, but you probably want to set this to False in a production
