--- airflow-scheduler.service.orig	2018-07-12 12:29:41.769099867 +0000
+++ airflow-scheduler.service	2018-07-12 12:30:03.412434559 +0000
@@ -4,6 +4,7 @@
 After=network-online.target
 
 [Service]
+LimitNOFILE=1048576
 EnvironmentFile=/etc/sysconfig/airflow
 Environment=
 Restart=on-failure
@@ -12,4 +13,4 @@
 KillSignal=SIGINT
 
 [Install]
-WantedBy=multi-user.target
\ No newline at end of file
+WantedBy=multi-user.target
